modules/solid_mechanics/src/materials/IsotropicPlasticity.C

```diff
@@ -38,10 +40,14 @@ IsotropicPlasticity::IsotropicPlasticity( const InputParameters & parameters)
    _hardening_variable(declareProperty<Real>("hardening_variable")),
    _hardening_variable_old(declarePropertyOld<Real>("hardening_variable"))
 {
-  if (_yield_stress <= 0)
+  if (isParamValid("yield_stress") && _yield_stress <= 0)
```

Comment:
Just a reminder that this line of code can be removed if you use the `InputParameters::addRangeCheckedParam()`